aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConfiguration.java

```diff
@@ -512,6 +512,20 @@ public Builder syncExpression(@NonNull String modelName,
             return Builder.this;
         }
 
+        /**
+         * Sets sync expressions for models to filter which data is synced locally.
+         * The expression is evaluated each time DataStore is started.
+         * The QueryPredicate is applied on both sync and subscriptions.
+         * [NOTE: this will override the previous syncExpression configuration if there's any]
+         * @param syncExpressions the Map of {@link DataStoreSyncExpression}s to filter data
+         * @return Current builder
+         */
+        @NonNull
+        public Builder syncExpressions(@NonNull Map<String, DataStoreSyncExpression> syncExpressions) {
```

Comment:
This API change hasn't been reviewed, please remove

Reply:
Gotcha! I just removed this in my latest comment